src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/BuildUpToDateCheck.cs

```diff
@@ -93,7 +93,7 @@ private void Log(LogLevel level, string message, params object[] values)
             IProjectSystemOptions projectSystemOptions,
             ConfiguredProject configuredProject,
             Lazy<IFileTimestampCache> fileTimestampCache,
-            IProjectAsynchronousTasksService tasksService,
+            [Import(ExportContractNames.Scopes.ConfiguredProject)] IProjectAsynchronousTasksService tasksService,
```

Comment:
Does this mean that the Up to date check has been in the broken state as soon as we we introduced this parameter?